databases/week3/DataBase3.sql

```diff
@@ -0,0 +1,158 @@
+
+CREATE DATABASE HWdataWeek3;
+USE HWdataWeek3;
+
+CREATE TABLE Meal (
+ id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+ title VARCHAR(255) UNIQUE,
+ description_meal TEXT,
+ location_meal VARCHAR(255),
+ when_created DATETIME,
+ max_reservations INT, 
+ price DECIMAL(10,2) NOT NULL,
+ created_date DATE NOT NULL
+);
+
+CREATE TABLE Reservation (
+ id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+ number_of_guests INT NOT NULL,
+ meal_id INT NOT NULL,
+ created_date DATE,
+ contact_phonenumber VARCHAR(255),
+ contact_name VARCHAR(50) NOT NULL,
+ contact_email VARCHAR(255),
+ FOREIGN KEY (meal_id) REFERENCES Meal(id)
+);
+
+CREATE TABLE Review (
+ id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+ title VARCHAR(255),
+ description_reviev TEXT,
+ meal_id INT NOT NULL,
+ stars INT,
+ created_date DATE,
+ FOREIGN KEY (meal_id) REFERENCES Meal(id)
+);
+
+
+ALTER TABLE Meal CONVERT TO CHARACTER SET utf8;
+
+-- Get all meals
+SELECT * FROM Meal;
+
+-- Add a new meal
+INSERT INTO Meal (id, title, description_meal, location_meal, when_created, max_reservations,price,created_date)
+VALUES (1, 'Пряний курячий суп зі стегном з мангала', 'Насичений, пряний бульйон з гречаною локшиною, маринованним яйцем та грильованою куркою', "Ukraine", "2023-01-01", 2, 90.99, "2023-01-02");  
+
+INSERT INTO Meal (title, description_meal, location_meal, when_created, max_reservations,price,created_date)
+VALUES ('Козацький ланч', 'Пісний борщ з часниковими пампушками та пиріг пастуха з томленою свининою. Подаємо з салатом із ферментованих овочів.', "Ukraine", "2023-01-01", 1, 90.99, "2023-02-04");  
+
+INSERT INTO Meal (title, description_meal, location_meal, when_created, max_reservations,price,created_date)
+VALUES ('Rød grød med fløde', '4 Келихи пива.', "Ukraine", NULL, 2, 200.15, "2023-03-05");  
+
+INSERT INTO Meal (title, description_meal, location_meal, when_created, max_reservations,price,created_date)
+VALUES ('Rød ROOOO', '10 Келихи пива.', "Ukraine", NULL, 3, 2220.15, "2023-03-12");  
+INSERT INTO Meal (title, description_meal, location_meal, when_created, max_reservations,price,created_date)
+VALUES ('Rød grød med fløde 22221', '55 Келихи пива.', NULL, NULL, 4, 1.15, "2023-04-05");  
+INSERT INTO Meal (title, description_meal, location_meal, when_created, max_reservations,price,created_date)
+VALUES ('BOrsh', NULL, NULL, NULL, 5, 1500.15, "2022-03-05");  
+
+-- Get a meal with any id, fx 1
+SELECT * FROM Meal 
+Where id = 1;
```

Comment:
Keywords should be written in uppercase.

Reply:
+